wiki/People/Levi/en.md

```diff
@@ -0,0 +1,9 @@
+# Levi
+
+Levi, also known as Lekuru, is the creator and primary developer of the Titanic! project. 
```

Comment:
```suggestion
Levi, also known as Lekuru, is the creator and primary developer of the Titanic! project.
```

Reply:
always the space at the end... why

Reply:
You just made me learn a habbit I didn't knew I ever have.